src/main/java/cn/nukkit/console/NukkitConsole.java

```diff
@@ -26,6 +26,9 @@ protected boolean isRunning() {
 
     @Override
     protected void runCommand(String command) {
+        if (command.startsWith("/")) {
```

Comment:
This should be removed as it will cause issues with commands like worldedit that use double slashes.

Reply:
The commands are not run with a "/" when sent by a client, and it fails when it has a "/" in the console. One of the biggest gripes I have is the console fails on a command that you start with a /. I'm not even sure why we allow plugin commands to start with a slash...

Reply:
Would performing a second lookup in `dispatchCommand` if the sender is Console and the failed lookup starts with `/` work better?